projects/ui/src/shim.cds-core.scss

```diff
@@ -159,23 +159,7 @@
 
   /* Responsive Navigation */
 
-  --clr-sliding-panel-text-color: #{tokens.$cds-global-color-gray-700}; // var(--clr-color-neutral-700);
-  --clr-nav-background-color: #{tokens.$cds-global-color-gray-200}; // var(--clr-color-neutral-200);
-  --clr-responsive-nav-hover-bg: #{tokens.$cds-global-color-gray-0}; // var(--clr-color-neutral-0);
   --clr-responsive-nav-trigger-bg-color: #{tokens.$cds-global-color-gray-0}; // var(--clr-color-neutral-0);
```

Comment:
Why do we keep this one in the shim?

Reply:
The idea was to clean all, but the visual tests show that we still have work on the responsive nav - I will continue the removal of the shim of the responsive nav in this PR

Reply:
Shim is deleted in commit 74424f2